stylesheet.css

```diff
@@ -0,0 +1,103 @@
+header {
+  display: flex;
+  padding: 10px 10px 10px 10px;
+  justify-content: space-around;
+  align-items: center;
+  border: 2px solid lightgray;
+  background-color: white;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  margin-left: 0px;
+  margin-right: 0px;
+}
+
+.login h3,
+a {
+  cursor: pointer;
+  color: blue;
+  border: none;
+  font-size: 20px;
+}
+.main-container {
+  width: 100%;
+  display: flex;
+  gap: 10px;
+}
+.personalinfo {
```

Comment:
follow the same convention for class naming through out  i.e. kebab case

Reply:
resolved!